prac_07/myguitars.py

```diff
@@ -0,0 +1,50 @@
+class Guitar:

```

Comment:
What if you wanted to create another Guitar instance in another program? Wouldn't it be a better idea to have your guitar class in a separate file?